web_app/models/immersed_corrosion_models.py

```diff
@@ -43,3 +43,130 @@ def eval_material_loss(self, time):
         material_loss = (0.00006*self.p['C'] + 0.0008)*time + self.p['b']
 
         return material_loss
+
+
+class thermo_nutrient_variability_steel_corrosion_model(immersed_corrosion_model):
+    '''
+        @article{Kovalenko_2016, 
+        title={Long-term immersion corrosion of steel subject to large annual variations in seawater temperature and nutrient concentration}, 
+        volume={13}, 
+        ISSN={1744-8980}, 
+        url={http://dx.doi.org/10.1080/15732479.2016.1229797}, 
+        DOI={10.1080/15732479.2016.1229797}, 
+        number={8}, 
+        journal={Structure and Infrastructure Engineering}, 
+        publisher={Informa UK Limited}, 
+        author={Kovalenko, Roman and Melchers, Robert E. and Chernov, Boris}, 
+        year={2016}, 
+        month=sep, 
+        pages={978–987} }
+
+    '''
```

Comment:
Please use the normal form for referencing a paper.

Reply:
done